test/acceptance/tests.py

```diff
@@ -829,16 +858,6 @@ def peer_grade_exists():
             "Learner still not graded after {} additional attempts".format(max_attempts)
         )
 
-
-class FullWorkflowBaseTest(OpenAssessmentTest, FullWorkflowMixin):
```

Comment:
To allow reuse with the accessibility test, I got rid of FullWorkflowBaseTest and moved the methods directly into FullWorkflowMixin.